src/main/java/com/revature/security/extra/model/AuthorityName.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.revature.security.extra;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This file "AuthorityName.java" should be located in "com/revature/security/extra" directory, not in "/home/jenkins/workspace/erview-evaluations_PR_PR-57-THYF2QUBWGI6EMGO73GWDKCQJ5HMZBBLF35CQMQWKF6SM2G4YTPA/src/main/java/com/revature/security/extra/model". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1598)